Cargo.toml

```diff
@@ -46,3 +46,4 @@ panic = 'abort'
 # Temporarily lock pgp to commit (master branch at time of writing) because the currently release crate locks zeroize to =1.3
 #pgp = { git = "https://github.com/rpgp/rpgp.git", rev = "21081b6aaaaa5750ab937cfef30bae879a740d23" }
 pgp = { git = "https://github.com/tari-project/rpgp.git", rev = "32939dbe86565d5ede769a7907ec42dfdf353849" }
+liblmdb-sys = { git = "https://github.com/leet4tari/lmdb-rs.git", branch = "v0.2.3" }
```

Comment:
Could you PR your changes into https://github.com/tari-project/lmdb-rs/tree/tari-changes and update this to reference that branch?